lib/validate.ts

```diff
@@ -142,11 +142,8 @@ export async function validateExamples(
     if (o.pretty) {
       /* tslint:disable-next-line:no-console no-string-literal */
       console.log(`Validating "examples" and "x-ms-examples" in  ${specPath}:\n`)
-      const errors = getErrorsFromModelValidation(
-        validator.specInJson,
-        validator.specValidationResult,
-      )
-      errorsAddFileInfo(validator.specInJson, errors)
+      const errors = getErrorsFromModelValidation(validator.specValidationResult)
+      errorsAddFileInfo(errors)
```

Comment:
do we need this call here ? wont it be called already inside `getErrorsFromModelValidation` which calls the `processValidationErrors`